packages/nextra-theme-docs/src/flexsearch.js

```diff
@@ -34,7 +34,7 @@ const Item = ({
           {page}
         </div>
       ) : null}
-      <Link href={Router.basePath + href}>
+      <Link href={href}>
```

Comment:
put it here 
<img width="557" alt="image" src="https://user-images.githubusercontent.com/7361780/180619546-e0ff6bf3-2e53-4df3-99d0-3ab1da9fac44.png">
since no need to use `Router` and `router` from `useRouter`